src/test/java/com/trello/authorization/AuthorisationTest.java

```diff
@@ -0,0 +1,69 @@
+package com.trello.authorization;
+
+import com.trello.Waiters;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class AuthorisationTest {
+
+    @AfterMethod
+    public void signOffFromTrello() {
+        System.setProperty("webdriver.chrome.driver", "C://Users//Office//Downloads//chromedriver_win32 (1)//chromedriver87.exe");
```

Comment:
We don't need to set path to driver every time test finished. Please move this to @BeforeClass

Reply:
Will be resolved after the change of the framework's structure